neqo-http3/src/connection.rs

```diff
@@ -298,7 +316,28 @@ impl<E: Http3Events + Default, T: Http3Transaction, H: Http3Handler<E, T>>
         self.streams_have_data_to_send.insert(stream_id);
     }
 
+    pub fn set_resumption_settings(&mut self, settings: Vec<(HSettingType, u64)>) -> Res<()> {
```

Comment:
The interface to NSS and transport doesn't work like this.  It has a token that pickles all the necessary state.  It requires serialization/parsing, but I think that it is easier to conceptualize.

Reply:
This is the second time we have this confusion. Http3Connection does not implement an interface to the application. Http3Client does. Https3Client does the serialization, similar to the transport. PR #277 will pull out conn(neqo_transport::Connection) from Http3Connection into Http3Client (this is needed because of the Http3Server). To prepare for PR #27 I implemented the serialization/parsing in Http3Client.